templates/campaigns.html

```diff
@@ -83,6 +83,10 @@ <h4 class="modal-title" id="campaignModalLabel">New Campaign</h4>
               <input type="text" class="form-control" placeholder="http://192.168.1.1" id="url"/>
 	      <label class="control-label" for="url">Schedule: </label>
               <input type="text" class="form-control" id="launch_date"/>
+          <label class="control-label" for="senddelay">Min delay time: <i class="fa fa-question-circle" data-toggle="tooltip" data-placement="right" title="Minimum randomized delay time between sending of each campaign email"></i></label>
```

Comment:
OK I may need some help here.  When I created the lines:
`
smtp_min_delay: $("#smtp_min_delay").val(),
smtp_max_delay: $("#smtp_max_delay").val(),
`

In the campaigns = {} structure in campaigns.js, then when I press the launch button I get:

Invalid JSON Structure

So I have traced that (I think) to the function 'API_Campaigns' in api.go but don't know what to do from there.  I have already defined the smtp delays in the campaign struct in go so I don't know why it complains.